contrib/hawq-ambari-plugin/src/main/resources/utils/add-hawq.py

```diff
@@ -0,0 +1,498 @@
+      repos_to_add):
+    raise Exception(
+        'Failed to update repository information on existing cluster, {0} with stack {1}-{2}'.format(cluster_name,
+                                                                                                     stack_name,
+                                                                                                     stack_version))
+
+  print 'INFO: Repositories are available on existing cluster, {0} with stack {1}-{2}'.format(cluster_name,
+                                                                                              stack_name,
+                                                                                              stack_version)
+
+
+def write_service_info(stack_dir):
+  """
+  Writes the service info content to the specified stack_dir
+  """
+  stack_services = os.path.join(stack_dir, 'services')
+
+  for service in ('HAWQ', 'PXF'):
+    source_directory = os.path.join(HAWQ_LIB_STAGING_DIR, service)
+    destination_directory = os.path.join(stack_services, service)
+
+    if not os.path.exists(source_directory):
+      raise Exception('{0} directory was not found under {1}'.format(service, HAWQ_LIB_STAGING_DIR))
+
+    service_exists = False
+    if os.path.exists(destination_directory):
+      service_exists = True
+      shutil.rmtree(destination_directory)
+
+    if service_exists:
+      print 'INFO: Updating service {0}, which already exists under {1}'.format(service, stack_services)
+
+    shutil.copytree(source_directory, destination_directory)
+
+    print 'INFO: {0} directory was successfully {1}d under directory {2}'.format(service,
+                                                                                 'update' if service_exists else 'create',
+                                                                                 stack_services)
+
+
+def build_parser():
+  """
+  Builds the parser required for parsing user inputs from command line
+  """
+  parser = OptionParser(
+      usage='Usage: ./add-hawq.py --user admin --password admin --stack HDP-2.4 --hawqrepo http://my.host.address/hawq-2.0.1.0/ --addonsrepo http://my.host.address/hawq-add-ons-2.0.1.0/')
+  parser.add_option('-u', '--user', dest='user', help='Ambari login username (Required)')
+  parser.add_option('-p', '--password', dest='password',
+                    help='Ambari login password. Providing password through command line is not recommended.\n'
+                         'The script prompts for the password.')
+  parser.add_option('-s', '--stack', dest='stack', help='Stack Name and Version to be added.'
+                                                        '(Eg: HDP-2.4 or HDP-2.5)')
+  parser.add_option('-r', '--hawqrepo', dest='hawqrepo', help='Repository URL which points to the HAWQ packages')
+  parser.add_option('-a', '--addonsrepo', dest='addonsrepo',
+                    help='Repository URL which points to the HAWQ Add Ons packages')
```

Comment:
What would the --version do?

Does it show the version of the script? Or is it an input parameter?